python/ray/services.py

```diff
@@ -1046,8 +1046,9 @@ def start_raylet(redis_address,
     else:
         java_worker_command = ""
 
-    # Create the command that the Raylet will use to start workers.
-    start_worker_command = ("{} {} "
+    # Create the command that the Raylet will use to start workers. We use -u
+    # to prevent buffering of worker stdout/stderr.
+    start_worker_command = ("{} -u {} "
```

Comment:
Does anyone anticipate any performance issues related to making the stdout/stderr totally unbuffered?

Reply:
Character buffered mode will definitely be a performance issue if there are even moderate amounts of logs.

What about `stdbuf -oL python` or something instead? Is this even the right fix?